demo/api/src/db/fixtures/generators/page-content-block-fixture.service.ts

```diff
@@ -27,7 +28,7 @@ import { ProductListBlockFixtureService } from "./blocks/text-and-content/produc
 import { StandaloneHeadingBlockFixtureService } from "./blocks/text-and-content/standalone-heading-block-fixture.service";
 import { TextImageBlockFixtureService } from "./blocks/text-and-content/text-image-block-fixture.service";
 
-export type BlockCategory = "layout" | "media" | "navigation" | "teaser" | "textAndContent";
+export type BlockCategory = "layout" | "media" | "navigation" | "teaser" | "textAndContent" | "form";
```

Comment:
Should we now generate a page for "form" as well?

https://github.com/vivid-planet/comet/blob/f2b6cded34bdfa21fc72261843b6647221f4e562/demo/api/src/db/fixtures/fixtures.command.ts#L120-L125